peekingduck/runner.py

```diff
@@ -117,6 +121,11 @@ def run(self) -> None:
 
                 outputs = node.run(inputs)
                 self.pipeline.data.update(outputs)
+                if num_iter == 0:
+                    node_end_time = perf_counter()
+                    self.logger.info(
+                        f"{node.name} init time = {node_end_time - node_start_time:.2f} sec"
+                    )
```

Comment:
There's a lot more information in the logs now, I'm thinking perhaps it'll be better to use a CLI flag to trigger if we want both the startup time and the init times to be shown? Logs will look cleaner.
```
2022-01-13 14:26:04 peekingduck.cli  INFO:  Startup time = 8.08 sec 
2022-01-13 14:26:04 peekingduck.runner  INFO:  Initialising peekingduck.pipeline.nodes.input.live... 
2022-01-13 14:26:04 peekingduck.runner  INFO:  peekingduck.pipeline.nodes.input.live init time = 0.03 sec 
2022-01-13 14:26:04 peekingduck.runner  INFO:  Initialising peekingduck.pipeline.nodes.model.posenet... 
2022-01-13 14:26:08 peekingduck.runner  INFO:  peekingduck.pipeline.nodes.model.posenet init time = 3.39 sec 
2022-01-13 14:26:08 peekingduck.runner  INFO:  Initialising peekingduck.pipeline.nodes.dabble.keypoints_to_3d_loc... 
2022-01-13 14:26:08 peekingduck.runner  INFO:  peekingduck.pipeline.nodes.dabble.keypoints_to_3d_loc init time = 0.00 sec 
2022-01-13 14:26:08 peekingduck.runner  INFO:  Initialising peekingduck.pipeline.nodes.dabble.check_nearby_objs... 
2022-01-13 14:26:08 peekingduck.runner  INFO:  peekingduck.pipeline.nodes.dabble.check_nearby_objs init time = 0.00 sec 
2022-01-13 14:26:08 peekingduck.runner  INFO:  Initialising peekingduck.pipeline.nodes.dabble.fps... 
2022-01-13 14:26:08 peekingduck.runner  INFO:  peekingduck.pipeline.nodes.dabble.fps init time = 0.00 sec 
2022-01-13 14:26:08 peekingduck.runner  INFO:  Initialising peekingduck.pipeline.nodes.draw.poses... 
2022-01-13 14:26:08 peekingduck.runner  INFO:  peekingduck.pipeline.nodes.draw.poses init time = 0.00 sec 
2022-01-13 14:26:08 peekingduck.runner  INFO:  Initialising peekingduck.pipeline.nodes.draw.tag... 
2022-01-13 14:26:08 peekingduck.runner  INFO:  peekingduck.pipeline.nodes.draw.tag init time = 0.00 sec 
2022-01-13 14:26:08 peekingduck.runner  INFO:  Initialising peekingduck.pipeline.nodes.draw.legend... 
2022-01-13 14:26:08 peekingduck.runner  INFO:  peekingduck.pipeline.nodes.draw.legend init time = 0.00 sec 
2022-01-13 14:26:08 peekingduck.runner  INFO:  Initialising peekingduck.pipeline.nodes.output.screen... 
2022-01-13 14:26:08 peekingduck.runner  INFO:  peekingduck.pipeline.nodes.output.screen init time = 0.06 sec 
2022-01-13 14:26:15 peekingduck.pipeline.nodes.dabble.fps  INFO:  Avg FPS over all processed frames: 7.76 
```

Reply:
Renamed node `init time` to node `setup time`, change `Initialising <node>` message to differentiate from the `Initialising` message during startup.